package.json

```diff
@@ -23,9 +23,10 @@
     "reconnecting-websocket": "^4.4.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "build-script": "node build-scripts/main.mjs",
+    "start": "npm run build-script & react-scripts start",
```

Comment:
Is it right with one ampersand here? In bash that runs the processes concurrently. But maybe not here?